src/i18n/el/translation.json

```diff
@@ -0,0 +1,333 @@
+﻿{
+  "translation": {
+    "downloadMobile": "Κάντε λήψη της έκδοσης για κινητές συσκευές:",
+    "about": "Σχετικά",
+    "time_to_complete": "Χρόνος για ολοκλήρωση",
+    "time_to_complete_hm": "Χρόνος για ολοκλήρωση: {{hours}} ώρες και {{minutes}} λεπτά",
+    "hours": "ώρες",
+    "please": "Επιλέξτε",
+    "login": "Σύνδεση",
+    "or": "ή",
+    "singUp": "Εγγραφή",
+    "mobileOnly": "Μόνο για κινητά",
+    "start": "Έναρξη",
+    "startTimedActivity": "Έναρξη χρονομετρημένης δραστηριότητας",
+    "continue": "Συνέχεια",
+    "save_and_exit": "Αποθήκευση και έξοδος",
+    "optional": "Προαιρετικό",
+    "pleaseAnswerTheQuestion": "Απαντήστε στην ερώτηση.",
+    "pleaseProvideAdditionalText": "Δώστε επιπλέον κείμενο",
+    "pleaseListenToAudio": "Ακούστε τον ήχο μέχρι το τέλος.",
+    "onlyNumbersAllowed": "Επιτρέπονται μόνο αριθμοί",
+    "answerTooLarge": "Δώστε μια πιο σύντομη απάντηση",
+    "questionCount_one": "{{count}} ερώτηση",
+    "questionCount_other": "{{count}} ερωτήσεις",
+    "timedActivityTitle": "αποτελεί χρονομετρημένη δραστηριότητα.",
+    "youWillHaveToCompleteIt": "Θα έχετε {{hours}} ώρες και {{minutes}} λεπτά για να το ολοκληρώσετε.",
+    "yourWorkWillBeSubmitted": "Η εργασία σας θα υποβληθεί αυτόματα όταν τελειώσει ο χρόνος.",
+    "countOfCompletedQuestions_one": "Ολοκληρώθηκε {{countOfCompletedQuestions}} από {{count}} ερώτηση",
+    "countOfCompletedQuestions_other": "Ολοκληρώθηκαν {{countOfCompletedQuestions}} από {{count}} ερωτήσεις",
+    "activityFlowLength_one": "{{count}} δραστηριότητα",
+    "activityFlowLength_other": "{{count}} δραστηριότητες",
+    "countOfCompletedActivities_one": "Ολοκληρώθηκε {{countOfCompletedActivities}} από {{count}} δραστηριότητα",
+    "countOfCompletedActivities_other": "Ολοκληρώθηκαν {{countOfCompletedActivities}} από {{count}} δραστηριότητες",
+    "pleaseCompleteOnThe": "Ολοκληρώστε τα",
+    "mindloggerMobileApp": "στην εφαρμογή MindLogger για κινητά",
+    "mustBeCompletedUsingMobileApp": "Χρησιμοποιήστε την εφαρμογή MindLogger για κινητά",
+    "completeUsingAppNow": "Αποκτήστε την εφαρμογή τώρα",
+    "common_loading_error": "Κάτι πήγε λάθος κατά τη φόρτωση δεδομένων :(",
+    "toast": {
+      "answers_submitted": "<strong>Ολοκληρώθηκε!</strong> Λάβαμε τις απαντήσεις σας.",
+      "next_activity": "<strong>Ολοκληρώθηκε!</strong> Ας περάσουμε στην επόμενη δραστηριότητα."
+    },
+    "applet_list_component": {
+      "noon": "Μεσημέρι",
+      "midnight": "Μεσάνυχτα"
+    },
+    "wrondLinkParametrError": "Φτάσατε σε αυτήν τη διεύθυνση URL κατά λάθος. Επικοινωνήστε με τον υπεύθυνο αυτής της βοηθητικής εφαρμογής για περαιτέρω βοήθεια.",
+    "activity_due_date": {
+      "available_all_day": "Διαθέσιμο: Όλη την ημέρα",
+      "scheduled_at": "Scheduled At",
+      "available": "Διαθέσιμο από",
+      "to": "έως",
+      "day": "day",
+      "days": "days",
```

Comment:
Are these the same in Greek?

Reply:
Thanks for taking a look at these translations @LashaunnaS! The translation file was provided with instructions on a few items which didn't require translation due to being legacy or unused strings. I'll update the PR description for clarity.